build/common.sh

```diff
@@ -541,6 +538,11 @@ function kube::build::clean_images() {
 }
 
 function kube::build::ensure_data_container() {
+  # This is temporary, while last remnants of _workspace are obliterated
```

Comment:
I'll beef up the comments.

  # This is temporary, while last remnants of _workspace are obliterated.  If
  # the data container exists it might be from before the change from
  # Godeps/_workspace/ to vendor/, and thereby still have a Godeps/_workspace/
  # directory in it, which trips up godep (yay godep!).  Once we are confident
  # that this has run ~everywhere we care about, we can remove this.
  # TODO(thockin): remove this after v1.3 is cut.